### PR TITLE
Add content to pages[#3999]

### PR DESCRIPTION
**Pull request by admiral96**
Beautified index and added some twist on 404

### CODE REVIEW

File: app/templates/404.html [hunk 1]
```diff
@@ -3,5 +3,38 @@
 
 {% block error_content %}
     <h1>Error 404</h1>
-    <h3>The page you're looking for was not found. Please head back <a href="/">home</a>.</h3>
+    <h3>The page you're looking for was not found. You can head back <a href="/">home</a> or you can stay on this page and find then click the crazy box below.</h3>
+    <hr style="width:90%;" align="left">
+    <div id="container" style="height: 450px;">
+    	<div id="noli_me_tangere" class="crazy-box" onClick="show_popup()" onmouseover="randommove()">You didn't found me, did ya?</div>
```
Comment (mknecht) [inline app/templates/404.html]: didn't _find_ — you decline the first  verb already; the second stays in base form

File: app/templates/404.html [hunk 2]
```diff
@@ -3,5 +3,38 @@
...
+    		var noli_me_tangere = document.getElementById('noli_me_tangere');
+ 			newWidth = Math.floor(Math.random() * (maxWidth - width));
+ 			newHeight = Math.floor(Math.random() * (maxHeight - height));
+ 			noli_me_tangere.style.left = newWidth + "px";
+ 			if(newHeight > 150) {
+ 				noli_me_tangere.style.top = newHeight + "px";
+ 			} else {
+ 				noli_me_tangere.style.bottom = newHeight + "px";
+ 			}
+ 			times_hovered++;
```
Comment (mknecht) [inline app/templates/404.html:31]: where did you find this code?

File: app/templates/index.html [hunk 3]
```diff
@@ -2,8 +2,23 @@
...
 
 {% block content %}
-    <h1>DRC Web-Hook app is coming soon.</h1>
-    <hr>
-    <p><i>Note: Please see also <a href="https://www.youtube.com/watch?v=qkBsuUHCFWo">Mulungwishi Fertilizer Project</a></i>.</p>
+    <h1>Mulungwishi Web-Hook app is coming soon.</h1>
+    <center>
+    <div class="green-border-with-shadow grow">
+    	<h3>What?</h3><hr class="green-line">
+    	<p>It is an application that would response based on received sms from farmers which is processed from the <a href="https://start.engagespark.com" class="invisible-underline" target="_blank">EngageSPARK</a> platform.</p>
```
Comment (mknecht) [inline app/templates/index.html]: uppercase SMS please

File: app/templates/404.html (hunk 2 above)
Comment (admiral96) [inline app/templates/404.html:31]: I made it myself

Comment (admiral96) [inline app/templates/404.html:31]: mostly mine but a few are from stackoverflow

Comment (mknecht) [inline app/templates/404.html:31]: link please?

Comment (admiral96) [inline app/templates/404.html:31]: its not full snippet... i  just gathered ideas from different code
http://stackoverflow.com/questions/35328287/moving-a-div-by-radio-buttons-with-math-random-without-using-jquery
http://www.webdeveloper.com/forum/showthread.php?18460-Move-a-button-on-the-onMouseOver-event
They didn't work for me so I recomposed it myself...

File: app/templates/index.html (hunk 3 above)
Comment (mknecht) [inline app/templates/index.html]: and please never link to start, only to www.

Comment (mknecht) [inline app/templates/index.html]: and please write the company's name correctly

**Comment by mknecht** [comment]
please address the remaining test comments

**Comment by mknecht** [comment]
test -> text

**Pull request merged**